src/runtime_src/tools/scripts/apu_recipes/init-apu.service

```diff
@@ -0,0 +1,10 @@
+[Unit]
+Description=APU init Daemon
+
+[Service]
+ExecStart=/usr/bin/init-apu
+StandardOutput=journal+console
+MemoryLimit=1G
```

Comment:
Is MemoryLimit needed here?
If so, it should probably be less than 1G.

Reply:
Thanks @jeffli-xilinx for pointing it, memory limit is not needed. Made changes please approve